lib/internal/blob.js

```diff
@@ -131,7 +131,7 @@ class Blob {
    *   endings? : string,
    *   type? : string,
    * }} [options]
-   * @returns
+   * @returns {any}
```

Comment:
```suggestion
   * @constructs {Blob}
```

https://github.com/nodejs/node/commit/88dd9724b0722e93e572e7d4bc2f395b335e3ce3

(same commit for other `@returns` changes)

Maybe the commit can be cherry-picked without trouble now?